lang/cs/Org.Apache.REEF.Driver/Org.Apache.REEF.Driver.DotNet.csproj

```diff
@@ -21,12 +21,13 @@ under the License.
     <AssemblyName>Org.Apache.REEF.Driver</AssemblyName>
     <Description>Driver for REEF.NET</Description>
     <PackageTags>REEF Driver</PackageTags>
-    <TargetFrameworks>netstandard2.0;netcoreapp2.0;net46;net451</TargetFrameworks>
   </PropertyGroup>
-  <Import Project="..\build.DotNet.props" />
+  <Import Project="..\build.DotNetLibrary.props" />
   <ItemGroup>
     <PackageReference Include="protobuf-net" Version="$(ProtobufVersion)" />
     <PackageReference Include="Microsoft.Avro.Core" Version="$(AvroVersion)" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net451' Or '$(TargetGramework)' == 'net46'">
```

Comment:
Why do we need this condition?

Reply:
1) If you reference any framework libraries for .net core, it will generate warnings saying the library cannot be found or referenced. This conditional prevents those warnings
2) There are cases where you want to include a different version for the framework. I did this xunit.DotNet.props as the later version stopped supporting 4.5.1. So I include an earlier version of xunit for 4.5.1.

These were not needed previously because the libraries were not targeting .net core -- but this change, its needed. 

Also when we drop 4.5.1 and 4.6. This conditional can be removed.

Reply:
> Also when we drop 4.5.1 and 4.6. This conditional can be removed.

Doesn't .NET 4.6 implement .NET Standard 2.0? If so, we could drop these when we drop support for .NET 4.5, right? If so, we should have a JIRA for dropping 4.5, and mark these places with `<--TODO` comments linking back to that JIRA. That way, we will find them when we want to close the JIRA.

Reply:
.Net Standard supports .net461, so yes the plan would be to drop .net46 and .net451. I have created a JIRA and will mark these in my next code review. The JIRA is:
https://issues.apache.org/jira/browse/REEF-1888